app/helpers/application_helper.rb

```diff
@@ -14,18 +14,18 @@ def render_navigation_link(link)
   def submit_button(text = 'search.submit', options = {})
     "<div class='button input_button'>
```

Comment:
could this be moved into a view instead of invoking htmlsafe?

Reply:
It could but that blows this story out quite a bit ...